public/js/components/Editor.js

```diff
@@ -87,6 +87,29 @@ const Editor = React.createClass({
     });
 
     this.editor.on("gutterClick", this.onGutterClick);
+
+    // Disable APZ for source editors. It currently causes the line numbers to
+    // "tear off" and swim around on top of the content. Bug 1160601 tracks
+    // finding a solution that allows APZ to work with CodeMirror.
+    this.editor.getScrollerElement().addEventListener("wheel", ev => {
```

Comment:
I agree, let's keep the component clean and free of hacks. It's unfortunate that we have to do this.... but thanks a bunch for fixing it! I was noticing this and was wondering what was going on.

If you move this code to `util/editor.js` we'll merge this.